portfolio/src/main/webapp/script.js

```diff
@@ -16,13 +16,14 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  const facts =
+      ['I have lost 150lbs in my life time', 'I love running and hope to do a marathon soon'
```

Comment:
It might be easier to keep track of each item in this list if it's on its own line.

Most companies will usually have a style guide to help you format your code, and they'll usually cover a wide range of topics.

Here's Google's external JavaScript style guide as an example: https://google.github.io/styleguide/jsguide.html

And it recommends using clang-format for this kind of formatting: https://clang.llvm.org/docs/ClangFormat.html

Reply:
ok, I formatted the array neatly using the google style guide.

 I tried to install homebrew to install clang but had no luck. Then I realized you just wanted me to keep my code formatting consistent with the clang-format. Rather than actually downloading it.

Reply:
Looks good to me.  I'll do some research to see if I can figure out how to easily enable clang-format for the cloud shell editor.